clients/assessment_component/src/assessment/pages/SettingsPage/components/CategoryList.tsx

```
@@ -0,0 +1,106 @@
+                  </AccordionTrigger>
+                </div>
+              </div>
+              <AccordionContent className='pt-4 pb-2 space-y-5 border-t mt-2'>
+                {category.competencies.length === 0 ? (
+                  <p className='text-sm text-muted-foreground italic'>
+                    No competencies available yet.
+                  </p>
+                ) : (
+                  <div className='grid gap-4 sm:grid-cols-2'>
+                    {category.competencies.map((competency) => (
+                      <CompetencyItem competency={competency} categoryID={category.id} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)